### PR TITLE
Fix motion gettext

### PR DESCRIPTION
**Pull request by nrobert13**
Maintainer:  @mips171 (original Roger D <rogerdammit [at] gmail.com>)
Compile tested: (ARM, Linksys, OpenWrt version 22.03.2)

Description:
Package compile fails without.

### CODE REVIEW

File: multimedia/motion/Makefile [hunk 1]
```diff
@@ -8,12 +8,12 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=motion
-PKG_VERSION:=4.3.2
+PKG_VERSION:=4.5.0
```
Comment (1715173329) [inline multimedia/motion/Makefile:11]: Please split this change to separate commit.

Comment (nrobert13) [reply multimedia/motion/Makefile]: thanks for the hint. updated PR.
